README.md

```diff
@@ -1,374 +1,29 @@
-# asdf [![Build Status](https://travis-ci.org/asdf-vm/asdf.svg?branch=master)](https://travis-ci.org/asdf-vm/asdf)
+# asdf-vm [![Build Status](https://travis-ci.org/asdf-vm/asdf.svg?branch=master)](https://travis-ci.org/asdf-vm/asdf)
 
-### _extendable version manager_
+**Manage multiple runtimes with a single CLI tool**
 
-Supported languages include Ruby, Node.js, Elixir and [more][plugins].
-Supporting a new language is as simple as [this plugin API][plugin_api].
+asdf-vm is a CLI tool that can manage multiple language runtimes on a per-project basis. It is like gvm, nvm, rbenv & pyenv (and more) all in one!
 
-[plugins]: https://github.com/asdf-vm/asdf-plugins
-[plugin_api]: https://github.com/asdf-vm/asdf/blob/master/docs/plugins-create.md
 
-## INSTALLATION
+## Why use asdf-vm?
 
-If you're on macOS you can [Install using Homebrew](https://github.com/asdf-vm/asdf#homebrew-on-macos)
+- single CLI for multiple languages
+- consistent commands to manage all your languages
+- single `.tool-versions` config file per project
```

Comment:
Might also be nice to mention the support for other version files like `.ruby-version`, `.node-version`, etc...